docs/import/add_disaster.js

```diff
@@ -172,6 +169,45 @@ function createTd() {
   return $(document.createElement('td'));
 }
 
+/**
+ * A common update method that writes to local data and firestore based on
+ * a customizable version of the value of the input.
+ * @param {Object} event
+ * @param {string} property
+ * @param {Function} fxn how to read/transform the raw value from the DOM.
+ * @return {?Promise<void>} See updateLayersInFirestore doc
+ */
+function onUpdate(event, property, fxn) {
+  const input = $(event.target);
+  const index = input.parents('tr').children('.index-td').text();
```

Comment:
is this done anywhere else? Should we extract to a utility function?

Reply:
Extracted - not entirely sure it's worth it but maybe we'll use again.